test/inventory/test_loader.py

```diff
@@ -98,14 +141,26 @@ def test_walk_task_for_port_161(self):
         }
 
         inventory_record = InventoryRecord(**inventory_record_json)
+
         result = gen_walk_task(inventory_record)
 
         self.assertEqual("sc4snmp;192.68.0.1;walk", result["name"])
         self.assertEqual("splunk_connect_for_snmp.snmp.tasks.walk", result["task"])
         self.assertEqual("192.68.0.1", result["target"])
         self.assertEqual([], result["args"])
-        self.assertEqual({"address": "192.68.0.1", "profile": None}, result["kwargs"])
+        self.assertEqual(
+            {
+                "address": "192.68.0.1",
+                "profile": None,
+                "chain_of_tasks_expiry_time": chain_of_tasks_expiry_time,
```

Comment:
Can you add a test when `chain_of_tasks_expiry_time` is `None`?

Reply:
Here `chain_of_tasks_expiry_time ` will never be `None` because it is created [here](https://github.com/splunk/splunk-connect-for-snmp/blob/fix/expires-timeout-configuration/splunk_connect_for_snmp/common/task_generator.py#L81), where this variable is set from env variable which has default value of 60 ([here](https://github.com/splunk/splunk-connect-for-snmp/blob/fix/expires-timeout-configuration/splunk_connect_for_snmp/common/task_generator.py#L6C14-L6C14).

However, there is one function which returns `None` if there are no tasks in redbeat or if existing tasks don't have an `expires` variable and this is [get_chain_of_task_expiry](https://github.com/splunk/splunk-connect-for-snmp/blob/fix/expires-timeout-configuration/splunk_connect_for_snmp/customtaskmanager.py#L118-L123). This function is used [here](https://github.com/splunk/splunk-connect-for-snmp/blob/fix/expires-timeout-configuration/splunk_connect_for_snmp/customtaskmanager.py#L55) inside `did_expiry_time_change`. Unit tests of the `get_chain_of_task_expiry` can be found [here](https://github.com/splunk/splunk-connect-for-snmp/blob/fix/expires-timeout-configuration/test/other/test_custom_task_manager.py#L234-L256) and tests if the `did_expiry_time_change` where `did_expiry_time_change` will return `None` is [here](https://github.com/splunk/splunk-connect-for-snmp/blob/fix/expires-timeout-configuration/test/other/test_custom_task_manager.py#L292-L298)